js/viz/core/utils.js

```diff
@@ -444,6 +444,50 @@ export function getVizRangeObject(value) {
     }
 }
 
+export function normalizeArcParams(x, y, innerR, outerR, startAngle, endAngle) {
+    let isCircle;
+    let noArc = true;
+    const angleDiff = roundValue(endAngle, 3) - roundValue(startAngle, 3);
+    if(angleDiff) {
+        if((abs(angleDiff) % 360) === 0) {
+            startAngle = 0;
+            endAngle = 360;
+            isCircle = true;
+            endAngle -= 0.01;
+        }
+
+        if(startAngle > 360) {
```

Comment:
why do we need this if? Seem like we can just write `startAngle %= 360;`

Reply:
As for readability, I disagree.
what do we say that the angle is greater than the full rotation (360 degrees), or does the angle have the remainder of the division by 360

in the second case, they make me think that I need another remainder

Reply:
sorry i didn't catch. Text me directly plz